src/components/dashboard/main-content/course-enrollments/tests/CourseSection.test.jsx

```diff
@@ -32,18 +32,21 @@ const CARD_COMPONENT_BY_COURSE_STATUS = {
 };
 const TEST_ENTERPRISE_UUID = 'test-uuid';
 
-const CourseSectionWrapper = (props) => (
-  <AppContext.Provider value={{
-    enterpriseConfig: {
-      uuid: TEST_ENTERPRISE_UUID,
-    },
-  }}
-  >
-    <CourseSection
-      {...props}
-    />
-  </AppContext.Provider>
-);
+function CourseSectionWrapper(props) {
+  return (
+    // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
Similar question about being consistent on how we're handling this ESLint rule (in some test files, you're using useMemo, but in others like here, you're not).